### PR TITLE
Bug 1166612 - [Stingray] Hide filter when entering edit mode

### PR DESCRIPTION
**Pull request by dwi2**
https://bugzilla.mozilla.org/show_bug.cgi?id=1166612

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=44da488b048e1347468220c4d31ff57cbe8fc787

File: tv_apps/smart-home/js/home.js [hunk 1]
```diff
@@ -127,6 +127,12 @@
         that.edit.init(that.spatialNavigator, that.cardManager,
                        that.cardScrollable, that.folderScrollable);
         that.edit.on('arrange', that.onArrangeMode.bind(that));
+        that.edit.on('enter-edit-mode', function() {
```
Comment (rexboy7) [inline tv_apps/smart-home/js/home.js]: Can we send `edit` to `filterManager` and insert listeners there?

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0ba9bd2c30c1d2b5f9036a43118a4fae828e3076

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9b56be044988ea512608beae7aab6ec5196825d9

**Comment by autolander** [comment]
The pull request could not be integrated in as it contained a commit which was missing a bug number. Please ensure all commits contain bug numbers, and are in the format of: Bug [number] - [description]

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cd21f2b16fe17599983d04fcaa034cdb1c0dd824

**Pull request merged**